docs/docsite/rst/community/maintainers_guidelines.rst

```diff
@@ -119,14 +123,82 @@ Some other general guidelines to encourage contributors:
 * If somebody suggests a good idea, mention it or put a thumbs up.
 * After merging, thank the author and reviewers for their time and effort.
 
+.. _review_checklist:
+
+Reviewing bug reports
+----------------------
+
+When users report bugs, verify the behavior reported. Remember to always be kind with your feedback.
+
+* [ ] Did the user made a mistake in the example code? We often see user errors reported as bugs.
+* [ ] Did the user assume an unexpected behavior? Ensure that the related documentation is clear.
```

Comment:
This bugs are useful to help us improve documentation

Reply:
SGTM changed, PTAL https://github.com/ansible/ansible/pull/77661#discussion_r860523050